ibm/config.go

```diff
@@ -594,91 +594,368 @@ func (sess clientSession) IAMNamespaceAPI() (*ns.IbmCloudFunctionsNamespaceAPIV1
 
 // CIS Zones Service
 func (sess clientSession) CisZonesV1ClientSession() (*ciszonesv1.ZonesV1, error) {
+	if sess.cisZonesErr == nil {
+		cisZonesV1Opt := &ciszonesv1.ZonesV1Options{
+			URL:           sess.cisZonesV1Client.Service.Options.URL,
+			Crn:           core.StringPtr(""),
+			Authenticator: sess.cisZonesV1Client.Service.Options.Authenticator,
+		}
+		cisZonesClient, cisZonesErr := ciszonesv1.NewZonesV1(cisZonesV1Opt)
+		if cisZonesErr != nil {
+			return nil, fmt.Errorf(
+				"Error occured while configuring CIS Zones client: %s",
+				cisZonesErr)
+		}
+		return cisZonesClient, nil
+	}
 	return sess.cisZonesV1Client, sess.cisZonesErr
```

Comment:
This looks good but if you can wait for a regen of the SDK it will be much simpler:

```suggestion
	return sess.cisZonesV1Client.Clone(), sess.cisZonesErr
```

Reply:
Thanks for your comments.  
I have upgraded sdk and regenerated cis sdk.  Modified the code as per your suggession.
Kindly review them.